server/controllers/getExercises/index.js

```diff
@@ -0,0 +1,27 @@
+const { getExercises } = require('../../database/queries');
+
+module.exports = async (req, res, next) => {
+  try {
+    const {
+      user: { id: userId },
+    } = req;
+    const [
+      {
+        rows: [{ weight }],
+      },
+      { rows: exercises },
+      { rows: userExercises },
+    ] = await getExercises(userId);
```

Comment:
If this function could return an object with three descriptive keys, instead of a three-length array, then there would be no chance of someone calling it and incorrectly expecting the order of the returned array.

Reply:
On second thought, it actually might make more sense for the controller to be responsible for calling three query functions, instead of having one query call which is actually doing three queries. That would keep the queries small and focused, and also allow for them to be reused in other files (where you may want just one query to be called, not all three.)